src/main/java/ch/admin/bag/covidcertificate/service/CovidCertificateGenerationService.java

```diff
@@ -30,33 +40,58 @@
     private final ObjectMapper objectMapper;
     private final CovidPdfCertificateGenerationService covidPdfCertificateGenerationService;
     private final CovidCertificateDtoMapperService covidCertificateDtoMapperService;
+    private final CovidCertificatePdfGenerateRequestDtoMapperService covidCertificatePdfGenerateRequestDtoMapperService;
+
+    public CovidCertificateCreateResponseDto generateFromExistingCovidCertificate(VaccinationCertificatePdfGenerateRequestDto pdfGenerateRequestDto) throws BarcodeException {
```

Comment:
Ist die Fall Unterscheidung zwischen Existierendem und Neuem notwendig? In der Theorie müsste ein Neuer dieselben Daten liefern wie ein Existierender, da das PDF identisch sein soll.